run-tests.php

```diff
@@ -1567,10 +1561,6 @@ function run_test($php, $file, $env)
 					show_result('SKIP', $tested, $tested_file, '', $temp_filenames);
 				}
 
-				if (isset($old_php)) {
-					$php = $old_php;
-				}
-
```

Comment:
Are you sure it is safe to delete this code (also the following occurrences)? It doesn't appear so to me, but I have a hard time understanding the code (run_test() has ~1000 LOC).